### PR TITLE
Fix/production issues

### PR DESCRIPTION
**Pull request by derrandz**

### CODE REVIEW

File: src/views/transaction-detail/index.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ class TransactionDetail extends Component {
 
     async getTx(txHash) {
         try {
-          const txResponse = await dataSource.getTx(txHash.toLowerCase()).then(({ transaction  }) => transaction);
+          const txResponse = await dataSource.getTx(txHash.toLowerCase());
 
             if (txResponse === undefined) {
                 console.log("Couldn't retrieve the transaction using the provided tx hash");
```
Comment (rem1niscence) [inline src/views/transaction-detail/index.js:86]: not from the PR but, is this supposed to be shown to the user or for internal use?

Comment (derrandz) [reply src/views/transaction-detail/index.js:86]: no clue, this is legacy code

**Review by nymd — APPROVED** [review]

File: src/views/transaction-detail/index.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ class TransactionDetail extends Component {
         super(props);
 
         this.state = {
-            txHash: this.props.location && this.props.location.data ? (this.props.location.data.txHash || {}) : {},
+            txHash: this.props.location && this.props.location.data ? this.props.location.data.txHash : undefined,
```
Comment (Evalir) [inline src/views/transaction-detail/index.js:27]: Hmmm this might blow up I think if `txHash` is undefined

**Review by Evalir — APPROVED** [review]
LGTM, just a comment

**Pull request merged**